narwhal/worker/src/metrics.rs

```diff
@@ -66,6 +66,20 @@ impl WorkerMetrics {
                 "created_batch_size",
                 "Size in bytes of the created batches",
                 &["epoch", "reason"],
+                // buckets with size in bytes
+                vec![
```

Comment:
Took the opportunity to introduce some bucketing for this metric as default bucketing (0..10) doesn't really help for our purpose. Our batches span in sizes of 100/1000ands